### DISCUSSION TITLE
Integrate Iconify icons pack

### DISCUSSION THREAD

**Discussion by hassnian** [kodadot/nft-gallery#10622] [Q&A]
added dark mode

<img src="https://github.com/user-attachments/assets/c5164866-97dc-4fdd-bfe0-ce74a3ebcf6b" width="300" />
<img src="https://github.com/user-attachments/assets/a1b282e3-8123-428e-b512-717886396f81" width="300" />

saw that these come from [iconify](https://iconify.design/) we should integrate this icons if we use them more 

ref:

- https://iconify.design/docs/iconify-icon/
- https://github.com/unplugin/unplugin-icons

_Originally posted by @hassnian in https://github.com/kodadot/nft-gallery/issues/10536#issuecomment-2226836449_

**Comment by vikiival** [comment]
What do we currently use? 
Font awesome ? 

Isn't easy to replace icons with iconify or something

cc @kodadot/internal-dev

**Comment by preschian** [comment]
sure, let's go~
https://github.com/kodadot/nft-gallery/issues/796#issuecomment-2078015535

**Reply by vikiival** [reply]
Can you please respond to @exezbcz in 
- https://github.com/kodadot/nft-gallery/issues/796#issuecomment-2078642152 ?

**Reply by preschian** [reply]
Oh, nothing at the moment. If we need something, we will ping @exezbcz for sure 👍🏻

> there was a import feature for the fontawesome, or that is at least what we did earlier

probably related to this stuff

**Comment by exezbcz** [comment]
there was a import feature for the fontawesome, or that is at least what we did earlier
